mk/serviceconfig.mk

```diff
@@ -28,6 +28,15 @@ configs/: .build-artefacts/last-version \
 	$(foreach topic, $(TOPICS), $(foreach lang, $(LANGS),curl -s --retry 3 --fail -o configs/${lang}/catalog.${topic}.json http:$(API_URL)/rest/services/$(topic)/CatalogServer?lang=$(lang) && jq . configs/${lang}/catalog.${topic}.json 1> /dev/null; ))
 
 
+meteoconfigs/: export METEO_TESTING_STYLE_BASEURL=$(METEO_STYLE_BASEURL)
+meteoconfigs/:  configs/
+	${PYTHON_CMD}  scripts/meteoStyleUrl.py
+
+.PHONY: s3uploadmeteoconfig
+s3uploadmeteoconfig: meteoconfigs/
```

Comment:
maybe these targets should somehow go into the repo bgdi-scripts? to have all data deployment related stuff together there?

Reply:
Don't know. We already have a `s3uploadmeteoconfig` rule and these custom meteo config will never ever be changed before _map.ge.admin.ch_ end of life-